nss-day-cohort-52/giffygram-flying-racoons
# Stars Button functions for saving and unsaving likes

#### Changes Made
2. Modified file `Postlist.js` to include star functionality.
3. Description of changes/modifications made.
Made an if and find function in click event listener to check if star is yellow or blank.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cb-markFaves2
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > What code to look for specifically and what should it be doing?
Look for if and find statement in postlist.js. When a blank star is clicked a like is added to the database and when a yellow star is clicked the like is deleted from the database.
    [ ] Does app function with change?
    
4. View code file.
    [ ] Confirm file modifications are present as indicated above.
    [ ] Confirm no unused code or extraneous comments exist.
